src/filters/truncate.js

```diff
@@ -22,7 +22,7 @@ angular.module('openshiftCommonUI')
         // Find the last word break, but don't look more than 10 characters back.
         // Make sure we show at least the first 5 characters.
         var startIndex = Math.max(4, charLimit - 10);
-        var lastSpace = truncated.lastIndexOf(/\s/, startIndex);
+        var lastSpace = truncated.lastIndexOf(' ', startIndex);
```

Comment:
It looks to me like this logic will run for even for strings smaller than `charLimit`, or am I wrong? We should check that small strings aren't truncated. That might have been masked before by the incorrect use of a regex for `lastIndexOf`.